src/Workspaces/Core/Portable/Storage/SQLite/v1/Interop/SafeSqliteHandle.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using System.Runtime.InteropServices;
+using SQLitePCL;
+
+namespace Microsoft.CodeAnalysis.SQLite.v1.Interop
+{
+    internal sealed class SafeSqliteHandle : SafeHandle
+    {
+        private readonly sqlite3 _wrapper;
+
+        public SafeSqliteHandle(sqlite3 handle)
+            : base(IntPtr.Zero, ownsHandle: true)
+        {
+            _wrapper = handle;
```

Comment:
consistify these?

Reply:
The base type defines a field `handle` which is exposed here. It's a weird situation all around because the type `sqlite3` really shouldn't exist except all the `raw` methods require it.

Reply:
This has now been addressed.